### PR TITLE
Player opacity0

### PR DESCRIPTION
**Pull request by cobo105322**
This option would be useful if you want others to see you, but not see others, for example at a particularly focus-heavy section.

### CODE REVIEW

File: CelesteNet.Client/Entities/Ghost.cs
Comment (0x0ade) [inline CelesteNet.Client/Entities/Ghost.cs:223]: Thanks for the pull request :) It'd be great if you could fix the spacing to match the rest of the code:

```suggestion
            if (CelesteNetClientModule.Settings.PlayerOpacity == 0) {
                Alpha = 0f;
            } else {
                Alpha = 0.875f * ((CelesteNetClientModule.Settings.PlayerOpacity + 2) / 6f);
            }
```

In the worst case I'll just fix the spacing myself before or after hitting the merge button.

**Review by 0x0ade — CHANGES_REQUESTED** [review]

**Comment by cobo105322** [comment]
Hey Jade, I commited your changes request for the spacing :)

**Review by 0x0ade — APPROVED** [review]
Thank you :)

**Pull request merged**
